plugins/arDominionB5Plugin/modules/actor/templates/_contextMenu.php

```diff
@@ -0,0 +1,41 @@
+<div class="sidebar-lowering">
+
+  <?php foreach ($lists as $list) { ?>
+
+    <?php $pager = $list['pager']; ?>
+
+    <section class="card sidebar-paginated-list mb-4"
+      data-total-pages="<?php echo $pager->getLastPage(); ?>"
+      data-url="<?php echo $list['dataUrl']; ?>">
+
+      <div class="card-header text-center p-2">
+        <h5 class="mb-0">
+          <?php echo __('%1% of', ['%1%' => $list['label']]); ?>
+          <span class="invisible float-end spinner">
```

Comment:
Not sure how problematic it may be not taking space in small widths with long headings, we may want to use flex layout if that's the case.

Reply:
Same about using `d-none`.

Reply:
Ok will do.